client/knowledge_base_search/src/__test__/containers/SearchContainer/FeatureSelect.test.jsx

```diff
@@ -11,59 +11,73 @@ describe('<FeatureSelect />', () => {
     selectedFeature: FeatureSelect.featureTypes.PASSAGES.value,
     isFetchingResults: false,
   };
+  const propsWithRelevancy = Object.assign({}, props, {
+    selectedFeature: FeatureSelect.featureTypes.TRAINED.value,
+  });
+
+  let passagesTab;
+  let relevancyTab;
+  let highlightedFeature;
+
+  function renderWithProps(propSet) {
```

Comment:
this is a useful abstraction, would be interesting to see if we can generalize it more into a helper function across all tests if we find this pattern is common